tests/test_setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eo pipefail
+
+# Usage: ./test_setup [CPU, GPU, Gaudi] [Deepspeed]
+
+# Step 1: Clone and create conda environment
```

Comment:
also we can print out those step instead of comments that can be checked in the CI log.

Reply:
added